# DetectedObjectArray.msg
# Array of detected objects

std_msgs/Header header
DetectedObject[] objects